components/library/qt5/Makefile

```diff
@@ -19,18 +19,19 @@ BUILD_BITS=		32_and_64
 
 include ../../../make-rules/shared-macros.mk
 
+COMPILER=gcc
 COMPONENT_NAME=		qt
-COMPONENT_VERSION=	5.8.0
-COMPONENT_VERSION_MJR=	5.8
-COMPONENT_REVISION=	12
+COMPONENT_VERSION=	5.12.9
+COMPONENT_VERSION_MJR=	5.12
+COMPONENT_REVISION=	13
 COMPONENT_FMRI=		library/qt5
```

Comment:
Don't increment COMPONENT_REVISON when you change COMPONENT_VERSION. Remove COMPONENT_REVISION instead.

Reply:
I see. I need a little help here with IPS dependency resolution: 
```
/data/oi-userland/components/library/qt5/build/manifest-i386-qt5.depend has unresolved dependency '
    depend type=require fmri=__TBD pkg.debug.depend.file=libEGL.so.1 \
        pkg.debug.depend.reason=usr/lib/qt/5.12/plugins/amd64/xcbglintegrations/libqxcb-egl-integration.so \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.path=lib/64 \
        pkg.debug.depend.path=usr/gcc/7/lib/amd64 \
        pkg.debug.depend.path=usr/lib/64 \
        pkg.debug.depend.path=usr/lib/qt/5.12/lib/amd64'.
```
i'm getting this even though we already specify `x11/library/mesa` to satisfy the dependency on GLX

Reply:
nvm i accidentally stripped the bypass directives from the old manifest

Reply:
Before resolving this problem, you'll need to run "gmake sample-manifest" and check whether manifests/sample-manifest.p5m has changed. If so, you'll need to incorporate the changes back into qt5.p5m (or re-generate this file if too many things have changed).
After that you should run "gmake REQUIRED_PACKAGES". This will generate the dependencies and you'll have to check/edit the Makefile. In many cases this will solve dependency problems.

Reply:
i ended up doing all of that manually because that.....wasn't really documented anywhere (!)

Reply:
Did you read http://docs.openindiana.org/dev/userland/ ?

Reply:
ah i'd been lurking here since _before_ the transition to MkDocs. thanks!